pkg/cri/store/container/status.go

```diff
@@ -97,6 +97,10 @@ type Status struct {
 	// Unknown indicates that the container status is not fully loaded.
 	// This field doesn't need to be checkpointed.
 	Unknown bool `json:"-"`
+	// LinuxContainerResources has container runtime resource constraints
```

Comment:
wdyt about using ... Resources            *ContainerResources here.. as was done in CRI...

Reply:
like the following?

```
type Status struct {
...
Resources *runtime.ContainerResources 
...
```

Yeah that makes sense. Will update the PR.